test/acceptance/specs/SendEmailVerificationSpec.scala

```diff
@@ -33,7 +34,8 @@ class SendEmailVerificationSpec extends BaseSpec
         save4LaterWithNoData("111111111")
         navigateTo(StartPage)
         verifyCurrentPage(StartPage)
-        authenticateGG("111111111")
+        authenticateGGUserAndReturnEoriEnrolment("111111111", "111111111")
```

Comment:
can we pass different value for `internalId` for differntiation purpose with eoriNumber value

Reply:
will be addressed as part of testing